templates/circleci.yml.erb

```diff
@@ -0,0 +1,53 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/<%= app_name %>
+    docker:
+      - image: circleci/ruby:<%= Suspenders::RUBY_VERSION %>-node
+        environment:
+          RAILS_ENV: test
+          PGHOST: localhost
+          PGUSER: <%= app_name %>
+      - image: postgres:10.5
+        environment:
+           POSTGRES_USER: <%= app_name %>
+           POSTGRES_DB: <%= app_name %>_test
+           POSTGRES_PASSWORD: ""
+    steps:
+      - checkout
+
+      - run:
+          name: Which Bundler?
+          command: bundle -v
+
+      - restore_cache:
+          keys:
+            - <%= app_name %>-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
```

Comment:
Does this need to be branch specific?

I kind of like the idea that this would restore the cache for any ol' branch so long as the gems are the same.